lib/modules/dosomething/dosomething_northstar/dosomething_northstar.module

```diff
@@ -157,6 +157,15 @@ function dosomething_northstar_get_northstar_user($drupal_id) {
   return $northstar_user_info;
 }
 
+function dosomething_northstar_get_user_by_term($term, $id) {
```

Comment:
If this isn't returning a user, perhaps `get_user_by_term` is poorly phrased? Maybe `user_exists_by_term`?